esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -299,36 +350,20 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
         IPropertyTree * customRequestTransform = nullptr;
         try
         {
-            customRequestTransform = target_cfg->queryPropTree("Method/xsdl:CustomRequestTransforms");
+            customRequestTransform = target_cfg->queryPropTree("xsdl:CustomRequestTransform");
```

Comment:
I know we talked about it, but if we think we're ever going to support xsdl:CustomRequestTransforms containers, we might as well code for it now